test/helper/passwordHelpers.test.js

```diff
@@ -0,0 +1,38 @@
+const chai = require('chai');
```

Comment:
Codacy found an issue: [Expected 1 empty line after require statement not followed by another require.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)